server.rb

```diff
@@ -46,6 +45,50 @@
 	}.to_json
 end
 
+# Get list of apps
+
+#seccat = app.details.secondary_category
+#firstsubseccat = app.details.secondary_first_sub_category
+#secondsubseccat = app.details.secondary_second_sub_category
+
+# Gets screenshots of live_version app
+get '/app/metadata' do
+	content_type :json
+	username = request.env['HTTP_USERNAME']
+	password = request.env['HTTP_PASSWORD']
+	bundle_id = params[:bundle_id]
+	client = Spaceship::Tunes.login(username, password)
+    client.team_id = request.env['HTTP_TEAM_ID']
```

Comment:
Let's read all the header in one place. Move this line after `password = request.env['HTTP_PASSWORD']`